pkg/git/git_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package git_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/shipwright-io/build/pkg/git"
+)
+
+var _ = Describe("Git", func() {
+	const (
```

Comment:
For these, how about switching to https://onsi.github.io/ginkgo/#extensions which would make it a lot easier to read?

Reply:
@xiujuan95 pls consider this one.

Reply:
@bigkevmcd thanks, this is a nice recommendation, we adopted it on this test case because of the simplicity it provides when writing specs.